serl_robot_infra/franka_env/envs/wrappers.py

```diff
@@ -202,14 +202,17 @@ def action(self, action: np.ndarray) -> np.ndarray:
             expert_a = np.concatenate((expert_a, gripper_action), axis=0)
 
         if time.time() - self.last_intervene < 0.5:
-            return expert_a
+            return expert_a, True
 
-        return action
+        return action, False
 
     def step(self, action):
-        new_action = self.action(action)
+
+        new_action, replaced = self.action(action)
```

Comment:
`replaced` is always False?

Reply:
It's set to true in line 205, which is when the current time is within 0.5 seconds of the previous nonzero spacemouse action (0.5 sec delay until the control is handed back to the policy)